Specifications/Language/5_Grammar/QSharpParser.g4

```diff
@@ -217,8 +218,8 @@ expression
     | expression '&&&' expression
     | expression '^^^' expression
     | expression '|||' expression
-    | expression 'and' expression
-    | expression 'or' expression
+    | expression ('&&' | 'and') expression
+    | expression ('||' | 'or') expression
```

Comment:
Because `&&` and `||` are new tokens, they need to be added to QSharpLexer.g4 also.